backend/scrumblr-api/api/app.js

```diff
@@ -1,19 +1,11 @@
-let response;
+const express = require('express');
 
-exports.lambdaHandler = async (event, context) => {
-    try {
-        // const ret = await axios(url);
-        response = {
-            'statusCode': 200,
-            'body': JSON.stringify({
-                message: 'hello world',
-                // location: ret.data.trim()
-            })
-        }
-    } catch (err) {
-        console.log(err);
-        return err;
-    }
+const app = express();
+//app.use(bodyParser.json());
```

Comment:
Commented code.

Reply:
Removed in the below commit. Thanks for noticing that